### ISSUE TITLE
Not working with canary Magisk (Zygisk)

### ISSUE THREAD

**Issue by SSadistic** [kdrag0n/safetynet-fix#110]

**Comment by rbits0** [comment]
An updated version that works with Zygisk is available on kdrag0n's Patreon https://kdrag0n.dev/patreon/safetynet-fix/v2.2.0

**Comment by SSadistic** [comment]
Thank you,but I'll pass...not that i don't want to donate,but really tight on money right now. Keep the good work.

**Comment by kdrag0n** [comment]
Sorry to hear that; please be patient in that case. I'm closing this issue because a Zygisk version is already available, as linked above.

**Comment by dhammel** [comment]
@kdrag0n 

I installed the version 2.2.0 with latest magisk canary, zygisk enabled on pixel 6, android 12 with Play store, play services and a safteynet test app enabled in denylist, this module enabled and after restart I can pass all including basic integrity, but CTS Profile has a mismatch Failed. The prior version, with riru I passed no problem on pixel 4 running android 11 and latest magisk stable.

May this be an issue specific to pixel 6 as it's fairly new? I'm still trying to figure out for sure if this denylist in magisk is an alternative to magisk hide, or not. Since if i don't use it, then everything will fail basic integrity.

I'm not sure of the best safteynet test app, as the one that used to be in magisk told me if it was basic or hardware attestation, and my current one doesn't display that.
